src/components/home/elements/TokenRecord.vue

```diff
@@ -315,15 +315,15 @@ export default {
         },
```

Comment:
set token balances to NaN before calling `this.chain.getTokenBalance(...)` (Line 307).

Reply:
not so good to do that. When network is slow, it will display empty string for a few moment after executing Token Transactions. It's better to keep showing the original balance and only show empty string when first log in. If network is not available, website will redirect to waring page.